dev/src/index.ts

```diff
@@ -204,17 +204,17 @@ export class Firestore {
   private _clientPool: ClientPool<GapicClient>|null = null;
 
   /**
-   * Whether the initialization settings can still be changed by invoking
-   * `settings()`.
+   * The configuration options for the GAPIC client.
    * @private
    */
-  private _settingsFrozen = false;
+  _settings: Settings = {};
 
   /**
-   * The configuration options for the GAPIC client.
+   * Whether the initialization settings can still be changed by invoking
```

Comment:
Nit: now that `initializationSettings` have been renamed to just `settings`, should the word "initialization" be removed here as well?

Reply:
I kind of prefer the explicitness here. Leaving as is.